src/java/fr/paris/lutece/plugins/mydashboard/modules/identity/web/EditIdentityXPage.java

```diff
@@ -182,6 +186,48 @@ public XPage getEditIdentityCoordinates( HttpServletRequest request ) throws Use
         return getXPage( TEMPLATE_GET_VIEW_MODIFY_IDENTITY, request.getLocale( ), model );
     }
     
+    @View( value = VIEW_CHECK_IDENTITY_COMPLETION )
+    public XPage checkIdentityCompletion( HttpServletRequest request ) throws UserNotSignedException
+    {
+        LuteceUser luteceUser = SecurityService.isAuthenticationEnable( ) ? SecurityService.getInstance( ).getRegisteredUser( request ) : null;
+
+        if ( luteceUser == null )
+        {
+            throw new UserNotSignedException( );
+        }
+        
+        if ( ( _dashboardIdentity == null ) || ( _dashboardIdentity.getConnectionId( ) == null ) || !_dashboardIdentity.getConnectionId( ).getValue( ).equals( luteceUser.getName( ) ) )

```

Comment:
is it really necessary ? 
how  "_dashboardIdentity.getConnectionId( ).getValue( )"  could be different from  "luteceUser.getName( ) "

Reply:
I used the same condition that is used elsewhere on the class to check if the dashboardIdentity is loaded.
I think this check is done in case of log out and log in with another account.